query/appservice/appservice_function_app_only_https_accessible.sql

```diff
@@ -0,0 +1,19 @@
+select
+  -- Required Column
+  app.id as resource,
+  case
+    when not https_only then 'ok'
```

Comment:
This case should throw alarm, please check once.
`Function App should only be accessible over HTTPS`

Reply:
updated the changes.